project_RL/sarsa/sarsa_lambda_agent.py

```diff
@@ -0,0 +1,56 @@
+import random
+
+import numpy as np
+
+
+class SarsaLambdaAgent:
```

Comment:
Sounds like a good idea. I think first everybody needs to have clear in mind what is an agent in this context, so that we can abstract and create the super class.